src/public/app/widgets/search_options/search_string.js

```diff
@@ -13,11 +13,11 @@ const TPL = `
     </td>
     <td class="button-column">
         <div class="dropdown help-dropdown">
-          <span class="bx bx-help-circle icon-action" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></span>
+          <span class="bx bx-help-circle icon-action" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></span>
```

Comment:
Small issue with the buttons on hover on dark mode, border disappears:

<img width="1023" alt="image" src="https://github.com/user-attachments/assets/b658cd99-3812-4f0f-80f0-347bacb6d42a">